datasets/machine-learning-dataset-for-sdo/README.md

```diff
@@ -60,6 +61,22 @@ The data is stored as follows:
 
 Each image is stored as a variable 'x' in each npz. For compression reasons, the data is stored as float16 if the data fits and float32 otherwise. You should immediately, however, convert the data to at least float32.
 
+
+### Indexing
+
+Using the `sdo_ml_indexer` utility the dataset can be indexed:
+
+```
+python sdo_ml_indexer.py --data_dir "/mnt/data02/sdo/stanford_machine_learning_dataset_for_sdo"  --target_dir "/mnt/data02/sdo/stanford_machine_learning_dataset_for_sdo_extracted" --extract True
+```
+
+resulting in an `index.csv` file in the root of the target directory
+
+```
+path,file_name,instrument,channel,timestamp
```

Comment:
these values should later be extended, e.g. by SDO position, GOES flux at this time, etc.